src/webextension/popup/unlock.js

```diff
@@ -0,0 +1,75 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
It might make sense to put this file in `src/webextension/unlock/index.js`, unless you have some plans in mind for sharing with the "view" popup.

Reply:
`src/webextension/unlock/index.js` was a holdover I thought I deleted.

Reply:
I think we'll want to have more popup views, per the work @changecourse is doing.

Reply:
Right; I don't know if we'll be sharing any compiled resources between them, so I'm not sure if there's any benefit to putting them all in one directory.